Base/ASBaseDefines.h

```diff
@@ -71,7 +71,7 @@
 #endif
 
 #ifndef ASDISPLAYNODE_WARN_DEPRECATED
-# define ASDISPLAYNODE_WARN_DEPRECATED 1
+# define ASDISPLAYNODE_WARN_DEPRECATED 0
```

Comment:
@maicki I think yes for now since they are disabled in Pinterest, and we are still actively working on the best way to do migrations.